src/containers/Welcome/index.js

```diff
@@ -20,30 +26,58 @@ export default class Welcome extends React.Component {
     }).isRequired,
   };
 
+  state = {
+    inputMarginTop: 104,
+    isFocused: false,
+  };
+
+  scrollUp = () => {
+    this.setState({ inputMarginTop: 75, isFocused: true });
```

Comment:
`inputMarginTop` can be calculated from a ternary of `isFocused`. We can either pass both margins as `props` or maybe better to pass an offset that is conditionally subtracted.